### DISCUSSION TITLE
Curator Deadlines - Nov 1st or sooner!

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/RiverscapesXML#301] [RAVE: Symbology/BusinessLogic] [answered]
Hello @Riverscapes/riverscape-project-curators,
The past five months we have made tremendous progress on getting all three flavors of RAVE working. Thank you all for your efforts. We have just gone in and updated all the milestones (i.e. deadlines) and assignments for the symbology and business logic parts of curation. We have a Riverscapes Monitoring Summit Nov 2-4, a BRAT Webinar Nov 10 and a Riverscapes Webinar Dec 1. These are hard deadlines. We have let slip our End of Year 1 BLM Riverscapes deadline of Oct 1. We got a lot of what we needed to get done most of the way there, but not all the way on some things. 

Please, go in and look at all your assignments in [RiverscapesXML](https://github.com/Riverscapes/RiverscapesXML/issues) and [RiverscapesData](https://github.com/Riverscapes/RiverscapesData/issues). Some of you have some in other repos too! Anyhow, new milestones are there not as a suggestion, but as an actual deadline. If those deadlines seem unreasonable, talk to me or @wally-mac. 

I expect each of you to be completely done checking that the project type you are curating is working consistently across ArcRAVE, WebRAVE and QRAVE. Post any issues as they arise and we'll deal with them. However, you should be checking all your assignments and leaving a trail of breadcrumbs to confirm that projects are done:
- Riverscapes Context
- Channel Area
- TauDEM
- VBET
- Confinement
- BRAT
- pyBRAT
- pyRCAT
- RVD

We're nearly there! Next steps... curation of reports and documentation. BTW, we have some great applicatns for the [Application Support Specialist and Technical Writer](https://careers-usu.icims.com/jobs/4297/application-support-specialist---riverscapes-consortium/job). 

Cheers,
Joe

**Comment by joewheaton** [comment]
Just a reminder, to check you zoom levels (lower left) for vector layers.  Make sure in checking webRAVE vector layers you zoom through the full range and report any things that need addressing.

**Reply by MattReimer** [reply]
Also there's a new indicator on the bottom left of the map to tell you what zoom level you're at (good for referencing in tickets)

**Comment by joewheaton** [comment, marked as answer]
You guys nailed this!
